lbry/wallet/transaction.py

```diff
@@ -803,6 +805,20 @@ async def create(cls, inputs: Iterable[Input], outputs: Iterable[Output],
 
         ledger, _ = cls.ensure_all_have_same_ledger_and_wallet(funding_accounts, change_account)
 
+        if everything and not any(map(lambda txi: not txi.txo_ref.txo.is_claim, tx._inputs)):
+            # Spend "everything" requested, but inputs not specified.
+            # Make a set of inputs from all funding accounts.
+            all_utxos = []
+            for acct in funding_accounts:
+                # TODO: Constraints for get_utxos()?
+                utxos = await acct.get_utxos()
```

Comment:
On the `jsonrpc_account_fund()` code path, it seems the only extraneous arg which could be interpreted as a constraint is `from_account=None`. Will investigate this...

Possibly this imposes a constraint to exclude claims?
https://github.com/lbryio/lbry-sdk/blob/dc427ecf6c71477fb01565c0843a35dfdf27b113/lbry/wallet/ledger.py#L958

Reply:
```
TXO_TYPES = {
    "other": 0,
    "stream": 1,
    "channel": 2,
    "support": 3,
    "purchase": 4,
    "collection": 5,
    "repost": 6,
}

    def constraint_spending_utxos(constraints):
        constraints['txo_type__in'] = (0, TXO_TYPES['purchase'])
```

More context:
https://github.com/lbryio/lbry-sdk/commit/7d333efd450b86ae0bb06cfe87871ba005432e98

I take it "other" is a generic txo carrying an amount of LBC created by wallet_send, and "purchase" is a purchase receipt txo created by purchase_create. It is unclear there is any non-zero amount associated with a purchase receipt txo. Is this correct? 

The intention of the original bug was to send only "other" funds, NOT purchase receipts (and NOT the claim types as you say). Transferring both "other" and "purchase" inside ```jsonrpc_account_fund()``` kind of makes sense because in that context both from/to accounts are in the same wallet.

I will work to exclude "purchase" txos from funds transferred by support_create, channel_create, stream_create, etc.